<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="pidLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PID: </string>
    </property>
   </widget>
   <widget class="QLabel" name="nicknameLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nickname:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>RÃ©ception</string>
       </property>
       <widget class="QListView" name="receptionView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>331</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Transimission</string>
       </property>
       <widget class="QListView" name="transmissionView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>331</width>
          <height>301</height>
         </rect>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <zorder>transmissionView</zorder>
       <zorder>groupBox_2</zorder>
       <zorder>groupBox_2</zorder>
       <zorder>groupBox_3</zorder>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Command input</string>
    </property>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="commandInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
